app/components/settings/categories/SupportSettings.scss

```diff
@@ -31,33 +31,16 @@
   button {
     cursor: pointer;
     letter-spacing: 1px;
+    text-decoration: underline;    
   }
 
   p {
     margin-bottom: 10px;
   }
 }
 
-
 :global(.YoroiClassic) .component {
   h1 {
     font-size: 16px;
   }
-
-  a,
-  button {
-    text-decoration: underline;
-  }
-}
-
-:global(.YoroiModern) .component {
-  h1 {
-    font-size: 18px;
```

Comment:
This means that we don't set a font-size at all and instead we're relying on either CSS from a parent for the font size or on the default. Probably we should avoid this.

Reply:
@SebastienGllmt 
Fixed: https://github.com/Emurgo/yoroi-frontend/pull/717/commits/c1a103c8771f5d1dcafbbeceae1759878b38d981